Mathlib/Algebra/BigOperators/Group/List.lean

```diff
@@ -205,7 +205,7 @@ theorem prod_take_mul_prod_drop (L : List M) (i : ℕ) :
 @[to_additive (attr := simp)]
 theorem prod_take_succ (L : List M) (i : ℕ) (p : i < L.length) :
     (L.take (i + 1)).prod = (L.take i).prod * L[i] := by
-  simp [take_succ, p]
+  simp [← take_concat_get', p]
```

Comment:
Do you have an idea why this change is necessary?

Reply:
`take_succ` and `Iff.symm take_concat_get'` are almost the same lemma. Thus removing the new global simp lemma `take_concat_get'` from the local simp set and leaving `take_succ` in there also works, but the old simp set looped.